site2/docs/io-develop.md

```diff
@@ -24,7 +24,7 @@ You can develop Pulsar source connectors and sink connectors.
 ### Source
 
 Developing a source connector is to implement the {@inject: github:`Source`:/pulsar-io/core/src/main/java/org/apache/pulsar/io/core/Source.java}
-interface, which means you need to implement the {@inject: github:`open`:/pulsar-io/core/src/main/java/org/apache/pulsar/io/core/Source.java#L33} method and the {@inject: github:`record`:/pulsar-functions/api-java/src/main/java/org/apache/pulsar/functions/api/Record.java#L28} method.
+interface, which means you need to implement the {@inject: github:`open`:/pulsar-io/core/src/main/java/org/apache/pulsar/io/core/Source.java#L33} method and the {@inject: github:`read`:/pulsar-io/core/src/main/java/org/apache/pulsar/io/core/Source.java#L41} method.
```

Comment:
```suggestion
interface, which means you need to implement the {@inject: github:`open`:/pulsar-io/core/src/main/java/org/apache/pulsar/io/core/Source.java#L34} method and the {@inject: github:`read`:/pulsar-io/core/src/main/java/org/apache/pulsar/io/core/Source.java#L42} method.
```

Reply:
Maybe we'd better remove the line indicator, because it might be changed. Such like:
```
interface, which means you need to implement the open method and read method in {@inject: github:`open`:/pulsar-io/core/src/main/java/org/apache/pulsar/io/core/Source.java}
```

Reply:
@Jennifer88huang @Anonymitaet please help take a look.

Reply:
ok. I have removed it from everywhere in this file.